package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "patio",
     "description": "Patio query engine and ORM",
-    "version": "4.0.0",
```

Comment:
Should this be a `5.0` release? The `pg-query-stream` seems like a significant update. I don't know enough about the backing library to know if there would be a breaking change.